radiography/src/main/java/radiography/compose/SlotTables.kt

```diff
@@ -0,0 +1,31 @@
+@file:JvmName("SlotTables")
+
+package radiography.compose
+
+import androidx.compose.runtime.SlotTable
+import androidx.ui.tooling.asTree
+import radiography.ViewFilter
+import radiography.ViewFilters
+import radiography.ViewStateRenderer
+import radiography.ViewStateRenderers
+import radiography.ViewTreeRenderingVisitor
+import radiography.renderTreeString
+
+/**
+ * Scans a particular [SlotTable], ie. an entire Compose tree. You probably want to use
+ * [Radiography.scan][radiography.Radiography.scan] instead.
+ */
+@ExperimentalRadiographyComposeApi
+@JvmSynthetic
+fun SlotTable.scan(
```

Comment:
what's a use case for using this public API directly? Since we're saying "you probably want scan instead", would be interesting to also mention why you'd want this.